library/std/src/os/windows/mod.rs

```diff
@@ -5,6 +5,22 @@
 //! the core `std` library. These extensions allow developers to use
 //! `std` types and idioms with Windows in a way that the normal
 //! platform-agnostic idioms would not normally support.
+//!
+//! # Examples
+//!
+//! ```no_run
+//! use std::fs::File;
+//! use std::os::windows::prelude::*;
```

Comment:
This will fail on any OS other than windows.
```suggestion
#![cfg_attr(windows, doc = "```no_run")]
#![cfg_attr(not(windows), doc = "```ignore")]
//! use std::fs::File;
//! use std::os::windows::prelude::*;
```

Reply:
Err, I just realized the whole `windows` module has `cfg(windows)`. So this should be fine.